x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -107,6 +111,7 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps> = ({
     setPopover(false);
   }, []);
   const [exceptionModalType, setOpenAddExceptionModal] = useState<ExceptionListType | null>(null);
+  const [openAddEventExceptionModal, setOpenAddEventExceptionModal] = useState<boolean>(false);
```

Comment:
Nit: it's good practice to prefix boolean variables with `is`, `has`, or `not`. I would suggest changing `openAddEventExceptionModal` to something like `isAddEventExceptionModalOpen`. Using `open` as a prefix makes the const look like an action, but in fact, it represents a piece of the component's state. Which makes it a bit misleading 🙂

Reply:
Totally agree with this, thanks!